controllers/config_manager.go

```diff
@@ -53,6 +54,7 @@ type KubeEnvoyConfigManager struct {
 	client.Client
 	Scheme       *runtime.Scheme
 	EnvoyManager *manager.EnvoyConfigManager
+	Validator    *validation.Proxy
```

Comment:
What do you think about making this an interface field?

Reply:
I'd say I'm a proponent of Proxy interface - once it's available as a standalone, deployable instance. We also don't have any kind of integration testing as of now. Right now I'm not 100% sure of the interface set so I'd leave it as it is - non-interface implementation - but it will be refactored into interface when it's separated from kusk operator.